examples/inline.html

```diff
@@ -0,0 +1,575 @@
+<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
+<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en">
+
+<head>
+	<link rel="stylesheet" href="https://ajax.googleapis.com/ajax/libs/jqueryui/1.12.1/themes/smoothness/jquery-ui.css">    
+	<script src="https://ajax.googleapis.com/ajax/libs/jquery/1.11.3/jquery.min.js"></script>
+    <script src="https://ajax.googleapis.com/ajax/libs/jquerymobile/1.4.5/jquery.mobile.min.js"></script>
+    <title>Car Pics 2.0 - 360 photos</title>
+</head>
+
+<style>
+    #frm2menu div:hover {
+        color: #428bca !important;
+    }
+</style>
+<div id="frm2image" class="carPicsSpinner" autospin="false" spinOnLoad ="false" spinSensitivity=2 carPicsUrl="http://api.carpics2p0.com/rest/spinner?dealer=001&vin=OUT001" style="touch-action: none;overflow:hidden;width :100vw ; height:75vw;border: none;position:absolute;z-index:10;background-color:#fff;">
+</div>
+
+<div id="georgecarmichael" class="carPicsSpinner"   carPicsUrl="http://api.carpics2p0.com/rest/spinner?dealer=001&vin=OUT001" style="margin-top:1500px;touch-action: none;overflow:hidden;width :960px ; height:720px;border: none;position:absolute;z-index:10;background-color:#fff;">
+</div>
+
+
+<div id="frm2image-sml" class="carPicsSpinner" autospin=true autospinDirection="left" spinOnLoad =true disableMouse=true carPicsUrl="http://api.carpics2p0.com/rest/spinner?dealer=001&vin=OUT001" numberOfConnections=4 id="frm2model-sml" style="margin-top:900px;touch-action: none;border: none;position:absolute; height:300px; width:400px; z-index:10;overflow:hidden;background-color:#fff;">
+</div>
+<div id="frm2image-sml2" class="carPicsSpinner" autospin=true autospinSleep=10 spinOnLoad =true disableMouse=true carPicsUrl="http://api.carpics2p0.com/rest/spinner?dealer=001&vin=OUT001"  id="frm2model-sml" style="touch-action: none;border: none;position:absolute;left:-1000px;top:-1000px; height:75px; width:100px; z-index:10;overflow:hidden;background-color:#fff;">
+</div>
+<script type="text/javascript">
+	$(document).on('mousemove', function(e){
+	    var htmel = document.getElementById("frm2image-sml2");
+	    htmel.style.top = e.originalEvent.pageY + 10 + "px";
+	    htmel.style.left = e.originalEvent.pageX +10 +  "px" ;
+	});
+</script>
+<script>
```

Comment:
This is intended - demonstrating intended way to inline code offers the client the opportunity to include javascript inline with other client code.  Allows them to package it with other code saving a http request.